test/harness.js

```diff
@@ -1,58 +0,0 @@
-/* Copyright 2020 Record Replay Inc. */
-
-// Harness used for local testing.
```

Comment:
Weird, i thought this file was used :)

Reply:
`/src/test/harness.js` *is* used, a lot. The thing that gave away this file not being used was the header include in it 😄 . I've never seen that in a JS file before!

Reply:
Wait... I'm confused, I think you're correct, the tests don't run without this file,  and also I do *not* understand how 

```
# include "header.js"
```

is valid JS? What am I missing here?

Reply:
Iirc, it's compiled to run tests in gecko